num2words/__init__.py

```diff
@@ -50,6 +51,7 @@
     'dk': lang_DK.Num2Word_DK(),
     'pt_BR': lang_PT_BR.Num2Word_PT_BR(),
     'he': lang_HE.Num2Word_HE()
```

Comment:
(I know, the best solution would be for a CI system to be plugged in this repo so we wouldn't have to do that manually...)